src/colvarproxy.cpp

```diff
@@ -141,6 +141,26 @@ int colvarproxy_system::get_molid(int &)
 }
 
 
+void colvarproxy_system::get_alch_lambda(cvm::real* lambda)
+{
+  cvm::error("Error in get_alch_lambda: alchemical lambda dynamics is not supported by this build.",
```

Comment:
Can you make these return the same `int` error code?

Reply:
Sure can!